src/Catalog/Helpers/FeedHelpers.cs

```diff
@@ -1,27 +1,27 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.Extensions.Logging;
-using Newtonsoft.Json.Linq;
-using NuGet.Services.Metadata.Catalog;
-using NuGet.Services.Metadata.Catalog.Persistence;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Xml.Linq;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using NuGet.Services.Metadata.Catalog.Persistence;
 
-namespace Ng.Jobs
+namespace NuGet.Services.Metadata.Catalog.Helpers
 {
     /// <summary>
-    /// Utility class to share functionality between Package2Catalog and Feed2Catalog.
+    /// Helper methods for accessing and parsing the gallery's V2 feed.
     /// </summary>
-    public static class CatalogUtility
+    public static class FeedHelpers
```

Comment:
The `Feed2Catalog` tests cover the existing code but I'll create some standalone unit tests.